pkg/cassandra/config/config.go

```diff
@@ -58,6 +58,24 @@ type Schema struct {
 	// while connecting to the Cassandra Cluster. This is useful for connecting to clusters, like Azure Cosmos DB,
 	// that do not support SnappyCompression.
 	DisableCompression bool `mapstructure:"disable_compression"`
+	//Datacenter is the name for network topology
+	Datacenter string `mapstructure:"datacenter" valid:"optional"`
+	// TraceTTL is Time To Live (TTL) for the trace data in seconds
+	TraceTTL int `mapstructure:"trace_ttl" valid:"optional"`
```

Comment:
can the type here be `time.Duration` so that the user could specify `72h`?

Reply:
I am assuming that there should be a validation to make sure that customer does specify something like "ms"?

Reply:
that's the point of using strong type - the business logic does not need to worry about validations, they should happen separately during parsing. I believe if you simply change `int` to `time.Duration` and use `10ms` as a value in YAML, the parser will do the right thing.

Reply:
the option is either we name the field `trace_ttl_seconds` and force the user to deal with ridiculous numbers like 1123200 (2w - can you tell? I can't without doing math), or we keep a clean name `trace_ttl` and let the user specify easy-to-read values like `24h` or `14d`

Reply:
Done. Also, added a validator for it.